CMakeLists.txt

```diff
@@ -532,6 +532,7 @@ if(CMAKE_HOST_SYSTEM_NAME MATCHES "Darwin")
 endif()
 find_package(BISON 3.0.4 REQUIRED)
 find_package(FLEX 2.5.13 REQUIRED)
+find_package(double-conversion 3.1.4 REQUIRED)
```

Comment:
That should always be satisfied as it's a dependency for folly right?

Reply:
Thats right, but I wanted to make this explicit as we directly depend on it, I can remove it if you feel strongly enough.

Reply:
No I agree!